components/compositing/compositor.rs

```diff
@@ -522,19 +522,28 @@ impl<Window: WindowMethods + ?Sized> IOCompositor<Window> {
                 Msg::WebDriverMouseButtonEvent(mouse_event_type, mouse_button, x, y),
                 ShutdownState::NotShuttingDown,
             ) => {
+                let dppx = self.device_pixels_per_page_px();
+                let point = dppx.transform_point(Point2D::new(x, y));
                 self.on_mouse_window_event_class(match mouse_event_type {
                     MouseEventType::Click => {
-                        MouseWindowEvent::Click(mouse_button, DevicePoint::new(x, y))
-                    },
-                    MouseEventType::MouseDown => {
-                        MouseWindowEvent::MouseDown(mouse_button, DevicePoint::new(x, y))
+                        MouseWindowEvent::Click(mouse_button, DevicePoint::new(point.x, point.y))
```

Comment:
dppx.transform_point returns a Point2D<f32, DevicePixel>, which is the same as DevicePoint. We should be able to pass `point` directly instead of calling `DevicePoint::new` in all of these branches.